Intersect.Server.Core/Entities/Player.cs

```diff
@@ -1347,20 +1394,39 @@ public void GiveExperience(long amount)
         }
     }
 
-    public void TakeExperience(long amount)
+    public void TakeExperience(long amount, bool enableLevelDown = false)
     {
-        if (this is Player && Options.Instance.Map.DisableExpLossInArenaMaps && Map.ZoneType == MapZone.Arena)
+        if (Options.Instance.Map.DisableExpLossInArenaMaps && Map.ZoneType == MapZone.Arena)
```

Comment:
Change this to `if (force || (Options.Instance.Map.DisableExpLossInArenaMaps && Map.ZoneType == MapZone.Arena))`

Reply:
Events taking experience should not be stopped by map type